### PR TITLE
Update fairship.sh from sndsw.sh, move to defaults-release, remove sndsw.sh and pull.yml

### PR DESCRIPTION
**Pull request by olantwin**
Finish work started in #76

### CODE REVIEW

File: fairship.sh [hunk 1]
```diff
@@ -25,7 +26,7 @@ incremental_recipe: |
   rsync -a $INSTALLROOT/*/*.h $INSTALLROOT/include
   #Get the current git hash
   cd $SOURCEDIR
-  FAIRSHIP_HASH=$(git rev-parse HEAD)
```
Comment (ThomasRuf) [inline fairship.sh:28]: Don't understand why removing FAIRSHIP_HASH against SNDSW. SNDSW/SND should not appear at all in this directory, I would have assumed.

Comment (olantwin) [reply fairship.sh]: Indeed, that was a mistake. Fixed now.

**Comment by ThomasRuf** [comment]
There is a problem with 
FairShip/master/0/online/ScalerUnpack.cxx:62:19: error: 'class FairRootManager' has no member named 'GetOutFile'; did you mean 'GetRootFile'?
    auto f = fMan->GetOutFile();
                   ^~~~~~~~~~
                   GetRootFile

For sndsw, I removed the online from build. Should we do the same for FairShip?

**Comment by olantwin** [comment]
This is due to a change in newer FairRoot versions, removing GetOutFile. The solution now is to use the "Sink" interface, which doesn't necessarily need to be a file.

See e.g. here:
https://github.com/FairRootGroup/FairRoot/blob/bee7c1258909d88556e8209d18feba488dfadaad/base/steer/FairRootManager.cxx#L1060=

I'm on holiday today, but will try to fix this when I'm back.

**Pull request merged**